java/src/main/java/ai/rapids/cudf/Table.java

```diff
@@ -475,6 +475,7 @@ private static native long writeORCFileBegin(String[] columnNames,
                                                int compression,
                                                int[] precisions,
                                                boolean[] isMapValues,
+                                               int stripe_size_rows,
```

Comment:
Let's keep naming convention in Java consistent.

```suggestion
                                               int stripeSizeRows,
```